web/src/app/components/Modal/EditModal.tsx

```diff
@@ -0,0 +1,32 @@
+import { Modal, Button, Box } from '@mantine/core';
+import { IconXboxX } from '@tabler/icons-react';
+import { ReactNode } from 'react';
+import styles from '../../styles/Modal.module.css';
+
+type ModalProp = {
+  opened: boolean;
+  close: () => void;
+  content: ReactNode;
+};
+
+export default function EditModal({ opened, close, content,title }: ModalProp) {
```

Comment:
The title prop is missing from your ModalProp object.

Reply:
There is no title for edit profile modal in Figma. Do you still wanna add it?